### DISCUSSION TITLE
Is this worth a pull request?

### DISCUSSION THREAD

**Discussion by NPap0** [imartinez/privateGPT#263] [Q&A]
I'm fairly new to this, but checking the code I see that we use the .env file
https://github.com/imartinez/privateGPT/blob/355b4be7c0972f71208251a14f47d739f8456fb5/ingest.py#L74
 
in the above line to get the source directory and if it is not there then we get the default value of source_documents.
Upon checking the .env file though I see that there is no setting/key for the source directory so it will always resort to the default value. 
Is my thinking correct and is it worth a pull request?

**Comment by thekit** [comment]
No this is covered in the installation notes in Readme.md

https://github.com/imartinez/privateGPT

`Rename example.env to .env and edit the variables appropriately.`

also if you're on windows there are a lot of prerequisites that don't get installed automatically so you'll have to do what I did to get them using pip, the relevant discussions are linked below

https://github.com/imartinez/privateGPT/discussions/227
https://github.com/imartinez/privateGPT/discussions/230
https://github.com/imartinez/privateGPT/discussions/233

**Reply by NPap0** [reply]
> No this is covered in the installation notes in Readme.md
> 
> https://github.com/imartinez/privateGPT
> 
> `Rename example.env to .env and edit the variables appropriately.`
> 
> also if you're on windows there are a lot of prerequisites that don't get installed automatically so you'll have to do what I did to get them using pip, the relevant discussions are linked below
> 
> #227 #230 #233

Hey there, are you sure we are talking about the same thing? I'm talking about maybe we are able to skip a few characters of code but your answer refers to something else? I don't have an issue with the .env file at all
